mizar/dp/mizar/operators/nets/nets_operator.py

```diff
@@ -134,3 +134,7 @@ def allocate_endpoint(self, ep):
     def deallocate_endpoint(self, ep):
         n = self.store.get_net(ep.net)
         n.deallocate_ip(ep.ip)
+
+    def process_remote_deployed_change(self, net, new):
```

Comment:
Please move this to a file common for all fornax related functions.

Reply:
Moved